common/artifacts/909_wc_artifacts_library_librams.txt

```diff
@@ -161,21 +153,18 @@ libram_of_righteousness_03 = {
 	indestructible = yes
 
 	learning = 3
-	combat_rating = 3
+	combat_rating = 15
 
-	global_revolt_risk = -0.006
-
-	light_group_opinion = 3
+	vassal_opinion = 10
```

Comment:
I'm ok with changing revolt risk to more CR and opinion, but pls change vassal opinion to religious opinion